test/Chainweb/Test/Pact/SPV.hs

```diff
@@ -458,6 +465,31 @@ createVerify code mdata time (TestBlockDb wdb pdb _c) _pidv sid tid bhe = do
+    go ref cid _bhe _bha _
+        | tid /= cid = return mempty
+        | otherwise = readIORef ref >>= \case
+            True -> return mempty
+            False -> do
+                -- q <- toJSON <$> createTransactionOutputProof_ wdb pdb tid sid bhe 0
+                cmd <- buildCwCmd $
+                  set cbSigners [mkSigner' sender00 []] $
+                  set cbCreationTime (toTxCreationTime time) $
+                  set cbChainId tid $
+                  mkCmd "0" $
+                  mkExec
+                    code
+                    -- TODO: this nesting seems redundant. Could we pass q just
+                    -- as literal string to verify-spv?
+                    (object [("proof",object [("proof", q)])])
```

Comment:
the nested `proof` property seems a bit odd

Reply:
Could the inner value of the outer `proof` property just be the literal string value `q` or does the types require it to be an object?